src/screens/dialogs/access-file-dialog.tsx

```diff
@@ -0,0 +1,80 @@
+import { useContext, useState, useEffect } from "react";
+import { SharedWithWhoService } from "../../services/files/get-users-file.service";
+import { Dialog } from "../../components/Dialog";
+import {
+  FilesDialogsContext,
+  AVAILABLE_DIALOGS,
+  AuthContext
+} from "../../context/index";
+
+export const AccessManagementDialog = () => {
+  const [usersWithAccess, setUsersWithAccess] = useState<string[]>([]);
+
+  const { dialogsVisibilityState, closeDialog, selectedFile } =
+    useContext(FilesDialogsContext);
+  const Open = dialogsVisibilityState[AVAILABLE_DIALOGS.ACCESS_MANAGEMENT];
+
+  const [_newAccess, setNewAccess] = useState(selectedFile?.uuid || "");
+
+  const { session } = useContext(AuthContext);
+
+  useEffect(() => {
+    const fetchUsersWithAccess = async () => {
+      const response = await SharedWithWhoService({
+        token: session?.token as string,
+        fileUUID: selectedFile?.uuid as string
+      });
+      if (response.success) {
+        setUsersWithAccess(response.users);
+      }
+    };
+
+    if (Open && selectedFile) {
+      fetchUsersWithAccess();
+    }
+  }, [Open, selectedFile, session]);
+
+  const handleSave = async () => {};
+
+  const handleUnshare = async () => {};
+
+  return (
+    <Dialog
+      isOpen={Open}
+      onClose={() => closeDialog(AVAILABLE_DIALOGS.ACCESS_MANAGEMENT)}
+      title="Manage Access"
+    >
+      <input
+        type="text"
+        placeholder="Enter a username to share with"
+        aria-label="Edit access permissions"
+        onChange={(e) => setNewAccess(e.target.value)}
```

Comment:
It seems that the `_newAccess` state is used to save a username with whom the user wants to share a file. So, why are you setting `selectedFile.uuid` as the default value?